packages/facebook/facebook_server.js

```diff
@@ -50,16 +50,32 @@ var getTokenResponse = function (query) {
 
   var responseContent;
   try {
-    // Request an access token
-    responseContent = HTTP.get(
-      "https://graph.facebook.com/v2.2/oauth/access_token", {
-        params: {
-          client_id: config.appId,
-          redirect_uri: OAuth._redirectUri('facebook', config),
-          client_secret: OAuth.openSecret(config.secret),
-          code: query.code
-        }
-      }).content;
+    // Request an access token, or request access token info if it was already given
+    // DDP clients should make a post request sending the accessToken before logging in with oauth credentialtoken/secret
```

Comment:
Refer to https://github.com/kenyee/android-ddp-client/blob/master/src/com/keysolutions/ddpclient/android/DDPStateSingleton.java

The purpose of this is to support logging in with OAuth for mobile clients. For mobile clients, the post request sending accessToken needs to be done before logging in. (Refer to loginWithOAuth and sendOAuthHTTPRequest)